user_docs/en/userGuide.md

```diff
@@ -3387,6 +3387,14 @@ It is very important to only install add-ons from sources you trust.
 The functionality of add-ons is unrestricted inside NVDA.
 This could include accessing your personal data or even the entire system.
 

```

Comment:
Add-ons are scanned by [Virus Total](https://www.virustotal.com/), however, this only detects known malware at submission time.